server/server.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config()
+
+const express          = require('express'),
+      port             = process.env.PORT,
+      path             = require('path'),
+      webpack          = require('webpack'),
+      config           = require('../webpack.config'),
+      compiler         = webpack(config),
+      cors             = require('cors')
+      app              = express(),
+      spotifyConnector = require('./spotify-connector'),
+      request          = require('request')
+      saveAcessToken   = require('./spotify-connector'),
+      Spotify          = require('spotify-web-api-js'),
+      spotifyApi       = new Spotify(),
+      XMLHttpRequest   = require('xmlhttprequest').XMLHttpRequest
+
+app.use(cors())
+app.options('*', cors())
+app.use(require('webpack-dev-middleware')(compiler, {
+  publicPath: config.output.publicPath
+}))
+
+saveAcessToken() // Returns accessToken
+
+app.get('/api/artist/:name', (req, res) => {
+  spotifyApi.searchArtists(req.params.name, {limit: 5}, (err, data) => {
+    console.log(data)
+    if (err) { console.log(err) }
+    else { res.send(data) }
```

Comment:
Nie do końca podoba mi się pisanie if/else w 1 linijce, ale to kwestia stylistyczna, więc jak chcesz :P ale poniżej masz już zapisane inaczej. Dobrze jest jednak trzymać się jednego stylu :)

Reply:
done, ten request tez rozbilem na lnijki